aws/resource_aws_ec2_local_gateway_route.go

```diff
@@ -0,0 +1,204 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+const (
+	ec2LocalGatewayRouteEventualConsistencyTimeout = 1 * time.Minute
+)
+
+func resourceAwsEc2LocalGatewayRoute() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsEc2LocalGatewayRouteCreate,
+		Read:   resourceAwsEc2LocalGatewayRouteRead,
+		Delete: resourceAwsEc2LocalGatewayRouteDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"destination_cidr_block": {
+				Type:     schema.TypeString,
```

Comment:
should we add a plan-time validation like `validateCIDRNetworkAddress` ?

Reply:
Cannnnnn do!